book/src/SUMMARY.md

```diff
@@ -33,7 +33,6 @@
     * [Custom Data Directories](./advanced-datadir.md)
     * [Validator Graffiti](./graffiti.md)
     * [Database Configuration](./advanced_database.md)
-	* [Local Testnets](./local-testnets.md)
```

Comment:
My only concern about removing this section is that it might break some links from external sites, however given the miniscule amount of content here I doubt anyone would bother linking to it directly :sweat_smile:

Reply:
Bad links happen, cleaner is more better, IMO :)